pkg/servicedisc/types.go

```diff
@@ -124,6 +124,7 @@ type Service struct {
 	Geo       *geo.LocationData `json:"geo,omitempty" gorm:"embedded"`
 	Version   string            `json:"version,omitempty"`
 	LocalIPs  pq.StringArray    `json:"local_ips,omitempty" gorm:"type:text[]"`
+	Info      interface{}       `json:"info,omitempty" gorm:"-"`
```

Comment:
Move 
```
type VPNInfo struct {
	Latency     float64
	Uptime      float64
	Connections int
}
```
here and use it in Info

Reply:
You're right. I don't know why but I was trying to solve it in hardway, lol.
Thanks, fixed.